README.md

```diff
@@ -44,46 +42,321 @@ see https://github.com/ioBroker/ioBroker/wiki/Installation
 
 After that the ioBroker should run and be available in browser under ```http://<ip>:8081/```
 
-### Start ioBroker controller on linux
+### Start ioBroker controller
+
+#### ... on Linux
 
 * run ```./iobroker start``` in the ioBroker directory to start the ioBroker controller in the background
-* watch the logfile ```tail -f log/iobroker.log```
+* watch the logfile ```tail -f log/iobroker.<Date>.log```
 
 or
 
 * run ```node node_modules/iobroker.js-controller/controller.js``` to start the ioBroker controller in foreground and watch the log on console
 
-### Start ioBroker controller on windows
+### ... on windows
 
 * run ```iobroker start``` in the ioBroker directory to start the ioBroker controller in the background console
 * check the logfile ```node_modules/iobroker.js-controller/log/iobroker.log```
 
 or
 
 * run ```node node_modules/iobroker.js-controller/controller.js``` to start the ioBroker controller in foreground and watch the log on console
-### Install Redis (Optional)
-- Linux from here: https://redis.io/download
-- Windows from here: https://github.com/MicrosoftArchive/redis/releases
+
+## Configuration
+
+The main configuration is stored in iobroker-data/iobroker.json
+
+## Feature Overview
 
 ### Admin UI
+**Feature Status: Stable**
+
+The admin adapter is installed automatically and starts a web-server that hosts the Admin UI. Default port is 8081, so just open http://&lt;iobroker-ip&gt;:8081/
 
-The admin adapter starts a web-server that hosts the Admin UI. Default port is 8081, so just open http://&lt;iobroker&gt;:8081/
 If port 8081 is occupied, you can install second Admin UI on alternate port and change port for first admin UI:
 
-* run ```./iobroker add admin --enabled --port 8090``` and go to the http://&lt;iobroker&gt;:8090/. Of course you can change port 8090 to other one.
+* run ```./iobroker add admin --enabled --port 8090``` and go to the http://&lt;iobroker&gt;:8090/. Of course you can change port 8090 to an other one.
 
-## Using REDIS as States-DB
-There is a possibility to use REDIS as states database. It is reasonable to do that for big installations or for systems with performance problems.
-It is possible to switch anytime between REDIS and In-Memory Javascript DB. The only problem, that all states must be updated by adapters again (values will be lost).
-Objects and configuration are not affected.
+### Command Line Interface
+**Feature Status: Stable**
+
+The command line interface is described at https://www.iobroker.net/#de/documentation/config/cli.md
+
+### Hostname
+**Feature Status: Stable**
+
+By default the hostname for the js-controller instance will be taken from the official hostname of the server the process runs on. The hostname should not change after the first start of ioBroker on this host.
```

Comment:
> The hostname should not change

Does this mean the user SHOULD not change the system's hostname or that the ioBroker hostname DOES not change after the first start?

Reply:
should not be changed after :-)